backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestResultUpload.java

```diff
@@ -0,0 +1,67 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import gov.cdc.usds.simplereport.db.model.auxiliary.UploadStatus;
+import gov.cdc.usds.simplereport.service.model.reportstream.FeedbackMessage;
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.hibernate.annotations.Type;
+
+@Getter
+@Setter
+@Entity
+@Slf4j
+@Table(name = "upload")
+public class TestResultUpload extends AuditedEntity {
+
+  @Column private UUID reportId;
+
+  @Column
+  @Type(type = "pg_enum")
+  @Enumerated(EnumType.STRING)
+  private UploadStatus status;
+
+  @Column private int recordsCount;
+
+  @ManyToOne(optional = false, fetch = FetchType.LAZY)
+  @JoinColumn(name = "org_id")
+  private Organization organization;
+
+  @Column()
+  @Type(type = "jsonb")
+  private FeedbackMessage[] warnings;
+
+  @Column()
+  @Type(type = "jsonb")
+  private FeedbackMessage[] errors;
+
+  protected TestResultUpload() {}
+
+  public TestResultUpload(
+      UUID reportId,
+      UploadStatus status,
+      int recordsCount,
+      Organization organization,
+      FeedbackMessage[] warnings,
+      FeedbackMessage[] errors) {
+    this.reportId = reportId;
+    this.status = status;
+    this.recordsCount = recordsCount;
+    this.organization = organization;
+    this.warnings = warnings;
+    this.errors = errors;
+  }
+
+  public TestResultUpload(UploadStatus status) {
```

Comment:
idk how i feel about this constructor, i know its used for when we dont reach reportstream, i would like to remove it, details of that below.